civiform/civiform
# Focus on question input when applicant clicks on question from review screen

### Description

Per accessibility recommendations, when an applicant clicks on a question from the application review screen, we want the focus to be set explicitly on the input for the question they clicked on. This PR adds that functionality.

### Checklist

#### General

- [x] Performed manual testing (only on Chrome because Firefox interacts poorly with tab order)

### Instructions for manual testing

1. Create and publish a program with multiple questions on a single screen
2. As an applicant, click "apply"
3. Turn on a screen reader
4. From the program summary screen, click on any question
5. The cursor should land you on the input for the question you clicked on and the screen reader should start reading from that question
6. You should also see two params in the url `questionName` and `questionType`

https://github.com/civiform/civiform/assets/24319951/5d471f5e-2ac2-4353-8a19-fa56f5fb56bd


### Issue(s) this completes

https://github.com/civiform/civiform/issues/3571
